src/main/java/hw6/entities/User.java

```diff
@@ -0,0 +1,9 @@
+package hw6.entities;
+
+import com.epam.jdi.tools.DataClass;
+
+public class User extends DataClass<User> {
```

Comment:
What is the reason of two similar classes here ? TestData  and User

Reply:
User is a class
TestData contains real data for test